lib/diagnostic.js

```diff
@@ -38,14 +46,35 @@ let owner = null;
 // Methods: vocalize
 
 // Replace null with your answer
-let pet = null;
+let pet = {name: 'Boozer',
+          speciess: 'Dog',
+          breed: 'Labrador',
+          vocalization: 'BuUZeR',
+
+          vocalize: function (name){
+            if (name === this.name){
+            return pet.vocalization;
+          }
+          }
+
+};
```

Comment:
It looks like you're trying to make vocalize run only if you call the pet by it's name? We're looking for something more simple:

```js
let pet = {
  name: 'Beau',
  species: 'Dog',
  breed: 'Lab',
  vocalization: 'bark',
  vocalize: function () {
    return this.vocalization;
  }
};
```